btdb.java

```diff
@@ -13,8 +13,8 @@
 public class btdb{
 	/**
 	* Main method for btdb class
-	* Passes the required files to the driver method
-	*
+	* Passes the required files to the driver method.
+	* @param args 	unused

```

Comment:
args is the one for the Data.bt and Data.values

Reply:
Oh, shall I remove it then?